fedoracommunity/search/index.py

```diff
@@ -307,16 +312,25 @@ def construct_package_dictionary(self, package):
         """
         package = copy.deepcopy(package)
 
-        name = package['name']
+
         try:
+            name = package['name']
             info = self.latest_active(name)
         except KeyError:
             log.warning("Failed to get pdc info for %r" % name)
             return
 
         # Getting summary and description
-        mdapi_pkg_url = '/'.join([self.mdapi_url, info['name'], 'srcpkg', info['global_component']])
-        meta_data = local.http.get(mdapi_pkg_url).json()
+        try:
+            mdapi_pkg_url = '/'.join([self.mdapi_url, info['name'], 'srcpkg', info['global_component']])
+            meta_data = local.http.get(mdapi_pkg_url).json()
```

Comment:
Do we want/Can we add a time-out to our requests?

Reply:
We are using the requests session object, so we could define a timeout. But the tricky question is how long should it be ?? 🤓

Reply:
We can start with something like 30 or 60 seconds and see how that goes.

Reply:
I ll do some test it my dev environment 😃